monkestation/code/datums/station_traits/neutral_traits.dm

```diff
@@ -12,3 +12,20 @@
 /datum/station_trait/announcement_duke/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/duke
+
+/datum/station_trait/new_space_law
+	name = "New Space Law"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 10
+	show_in_report = TRUE
+	report_message = "A recent bet at centcom has led to a new space law being created. A report on it will be printed at all security record consoles."
```

Comment:
Proper name
```suggestion
	report_message = "A recent bet at Central Command has led to a new space law being created. A report on it will be printed at all security record consoles."
```